pkg/apis/operator/v1alpha1/tektonchain_types.go

```diff
@@ -115,6 +115,7 @@ type ChainProperties struct {
 	KMSRef               string `json:"signers.kms.kmsref,omitempty"`
 	KMSAuthAddress       string `json:"signers.kms.auth.address,omitempty"`
 	KMSAuthToken         string `json:"signers.kms.auth.token,omitempty"`
+	KMSAuthTokenDir      string `json:"signers.kms.auth.token-dir,omitempty"`
```

Comment:
Can you also add the other new fields for mongo-server-url which we have added as well

Reply:
Added the fields. Please check and approve